src/main/java/com/vfasad/service/PrintService.java

```diff
@@ -0,0 +1,82 @@
+package com.vfasad.service;
+
+import com.vfasad.entity.Order;
+import com.vfasad.entity.OrderConsume;
+import com.vfasad.entity.OrderStatus;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+@Service
+public class PrintService {
+    final int FIRST_TABLE_ROW = 5;
+
+    @Value("classpath:templates/xlsx/ReportInProgress.xlsx")
+    private File xlsxReportTemplate;
+
+    @Autowired
+    private OrderService orderService;
+
+    public byte[] generateInProgressXlsx() {
+
+        try {
+            FileInputStream file = new FileInputStream(xlsxReportTemplate);
+            XSSFWorkbook workbook = new XSSFWorkbook(file);
```

Comment:
Maybe we can pass `file` as contructor argument here instead of FileInputStream? Please, check